beacon-chain/rpc/beacon/van_minimal_consensus_info.go

```diff
@@ -0,0 +1,56 @@
+package beacon
+
+import (
+	ptypes "github.com/gogo/protobuf/types"
+	types "github.com/prysmaticlabs/eth2-types"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/feed"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// StreamMinimalConsensusInfo to orchestrator client every single time an unconfirmed block is received by the beacon node.
+func (bs *Server) StreamMinimalConsensusInfo(
```

Comment:
@mxmar If any block is skipped in any slot then what will happen? In that case, this method does not send consensus info to orchestrator. Right? Before starting this feature implementation, i suggested you to follow `StreamDuties(req *ethpb.DutiesRequest, stream ethpb.BeaconNodeValidator_StreamDutiesServer)` api from `rpc/validator/assignments.go`.

Reply:
Refactored to epochTicker approach instead of events. Right now I'm checking `slot == endSlot` before sending consensus info.